test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -62,6 +62,39 @@ class SolidityExecutionFramework: public solidity::test::ExecutionFramework
 	) override
 	{
 		bytes bytecode = multiSourceCompileContract(_sourceCode, _sourceName, _contractName, _libraryAddresses);
+
+		// Note: Temporary solution that adds arguments size in the EOF data_section_size header to validate evmone implementation.
+		// TODO: Validate header and version.
+		if (bytecode.size() > 3)
+		{
+			if (bytecode[0] == uint8_t(0xef) && bytecode[1] == uint8_t(00)) // is EOF
+			{
+				const auto firstSectionOffset = 3; // magic + version
+				bool stop = false;
+				for (size_t i = firstSectionOffset; i < bytecode.size(); ++i)
```

Comment:
```suggestion
				for (size_t i = firstSectionOffset; i < bytecode.size() && !stop; ++i)
```
Should allow for getting rid of
```
if (stop)
    break;
```